### DISCUSSION TITLE
Defining a Shipping Method Cost

### DISCUSSION THREAD

**Discussion by joybcode** [duncanmcclean/simple-commerce#1011] [Help] [answered]
Hi I've just created a shipping method which created a php file. Where and how can I define the cost of this shipping method?

**Comment by duncanmcclean** [comment, marked as answer]
You can do it in the `calculateCost` method, [as per the docs](https://simple-commerce.duncanmcclean.com/shipping#content-building-custom-shipping-methods). For example, for a £5 shipping cost, you'd do this (you need to return the amount in pence/cents).

```php
public function calculateCost(Order $order): int
{
    return 500;
}
```

**Reply by joybcode** [reply]
That's great. Thanks
